packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -209,6 +237,9 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
                 menuIconProps={{ hidden: true }}
                 menuProps={{ items: moreButtonContextualMenuItems() }}
                 showLabel={!props.isMobile}
+                /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */ disabled={
+                  props.disableForHoldScreen
```

Comment:
i think the more button should only be disabled iff all child items are disabled (this will allow support for custom button injection into the more button to still function correctly)